src/main/java/blackjack/domain/player/Profit.java

```diff
@@ -0,0 +1,36 @@
+package blackjack.domain.player;
+
+import java.util.Objects;
+
+public class Profit {
+    private final int value;
+
+    public Profit(double value) {
+        this.value = (int) value;
+    }
+
+    public Profit subtract(Profit playerProfit) {
```

Comment:
```suggestion
    public Profit subtract(Profit other) {
```
playerProfit이란 용어만 보면 사용될 클라이언트가 정해져있는 느낌이에요.